utils/get_latest_copr_build

```diff
@@ -35,20 +36,16 @@ builds = client.build_proxy.get_list(
     projectname=projectname,
     packagename=packagename)
 
-if '--debug' not in sys.argv:
+if '--debug' in sys.argv:
     json.dump(builds, sys.stderr, sort_keys=True, indent=2)
 
 for build in builds:
     # Version in COPR contains VERSION-RELEASE string. We need just the
-    # release. Additionally we do not care about *elX* suffix. So remove it
-    # when compare.
-    _release = build['source_package']['version'].split("-")[-1]
-    # FIXME: instead of rsplit(.el8) remove whole elx.* as it can contain
-    # later e.g. .el8_0 instead of .el8
-    if _release.rstrip(".el8") == pkgrelease:
+    # release. Additionally we do not care about *el8* suffix.
+    _release = re.findall(r"^[0-9.]*?-(.*)\.el8.*?$", build)
```

Comment:
I don't see the reason here for this regex I would do it similar like it was before:

```python
_release = build['source_package']['version'].split('_')[-1]
if _release.startswith(pkgrelease):
```

And then it doesn't matter what the dist part is

Reply:
Ok, that makes sense and the code is shorter. Thanks.